timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.java

```diff
@@ -283,6 +283,57 @@ public void testExceptionLoggingUsingMessage() throws Exception {
         + "0 errors, 1 warnings\n");
   }
 
+  public void testExceptionLoggingUsingVariable() throws Exception {
+    @Language("JAVA")
+    String source2= ""
+        + "package foo;\n"
+        + "import timber.log.Timber;\n"
+        + "public class Example {\n"
+        + "  public void log() {\n"
+        + "     String msg = \"Hello\";\n"
+        + "     Exception e = new Exception();\n"
+        + "     Timber.d(e, msg);\n"
+        + "  }\n"
+        + "}";
+    assertThat(lintProject(java(source2), timberStub)).isEqualTo(NO_WARNINGS);
+  }
+
+  public void testExceptionLoggingUsingNull() throws Exception {
+    @Language("JAVA")
+    String source2= ""
+        + "package foo;\n"
+        + "import timber.log.Timber;\n"
+        + "public class Example {\n"
+        + "  public void log() {\n"
+        + "     Exception e = new Exception();\n"
+        + "     Timber.d(e, null);\n"
+        + "  }\n"
+        + "}";
+    assertThat(lintProject(java(source2), timberStub)).isEqualTo("src/foo/Example.java:6: "
+        + "Warning: Use single-argument log method instead of null/empty message [TimberExceptionLogging]\n"
+        + "     Timber.d(e, null);\n"
+        + "                 ~~~~\n"
+        + "0 errors, 1 warnings\n");
+  }
+
+  public void testExceptionLoggingUsingEmpty() throws Exception {
```

Comment:
Thanks, seemed I wasn't paying enough attention (the `source2` bits as well). I've cleaned it up.